scripts/AbinsModules/LoadCASTEP.py

```diff
@@ -0,0 +1,265 @@
+import numpy as np
```

Comment:
To think about. Could this be a candinate for a general Mantid Load algorithm. Recommand speak to Sanghamitra or Louise